smithy-jsonschema/src/main/java/software/amazon/smithy/jsonschema/JsonSchemaShapeVisitor.java

```diff
@@ -329,6 +333,10 @@ private Schema.Builder updateBuilder(Shape shape, Schema.Builder builder) {
             builder.defaultValue(shape.expectTrait(DefaultTrait.class).toNode());
         }
 
+        shape.getTrait(DeprecatedTrait.class)
```

Comment:
Let's have this use the same `if` syntax as above.

Reply:
[0ebe006](https://github.com/smithy-lang/smithy/pull/2221/commits/0ebe0063cbc7844e1d67cf07712306a84028b5cc)